check.sh

```diff
@@ -5,5 +5,5 @@
 # Run in the same directory
 cd "$(dirname "$(readlink -f "$0")")"
 
-xmllint --noout --schema tome.xsd CONstitution.xml
-xsltproc --noout tome.xsl CONstitution.xml && echo "xsltproc completed successfully"
+xmllint --noout --schema tome.xsd CONstitution.xml || exit 1
```

Comment:
I remember
```
#!/bin/bash
set -ex
```
at the top of a script from my bash days if you want any failure to result in the script exiting. I don't know if it's the -e or the -x which does that. Probably the x, since the e sounds like "echo".